apps/users/src/app/dtos/dto.ts

```diff
@@ -45,3 +45,9 @@ export class UpdateMobileDto {
   @IsString()
   mobileNumber: string;
 }
+
+export class UserSettingsDto {
+  @ApiProperty()
+  @IsArray()
```

Comment:
```suggestion
  @IsString({ each: true })
```
Use this instead if we want to validate the input type that is an array of strings.